### PR TITLE
UserPrecautionStatus enum implemeted

### PR DESCRIPTION
**Pull request by Serhii-Buniak**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.Resources/UserPrecautionStatus.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace EPlast.Resources
+{
+    public enum UserPrecautionStatus
+    {
+        [Description("Прийнято")]
+        Accepted,
+        [Description("Потверджено")]
```
Comment (ostappppppp) [inline EPlast/EPlast.Resources/UserPrecautionStatus.cs:9]: Does it should be like this `Потверджено` in Plast community or `Підтверджено` as in normal Ukrainian language?

File: EPlast/EPlast.WebApi/Models/UserPrecaution/UserPrecautionViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public class UserPrecautionViewModel
+    {
+        public int Id { get; set; }
+        public int PrecautionId { get; set; }
+        public PrecautionDTO Precaution { get; set; }
+        public string Reporter { get; set; }
+        public string Reason { get; set; }
+        [Required]
+        public UserPrecautionStatus? Status { get; set; }
```
Comment (ostappppppp) [inline EPlast/EPlast.WebApi/Models/UserPrecaution/UserPrecautionViewModel.cs:17]: If it's property is required then why it's marked as nullable, just a little bit confusing or there is some reason for that?

File: EPlast/EPlast.Resources/UserPrecautionStatus.cs (hunk 1 above)
Comment (Serhii-Buniak) [reply EPlast/EPlast.Resources/UserPrecautionStatus.cs:9]: `Потверджено` is not my idea. This word has been used everywhere in the project. We have this word in database tables

File: EPlast/EPlast.WebApi/Models/UserPrecaution/UserPrecautionViewModel.cs (hunk 2 above)
Comment (Serhii-Buniak) [reply EPlast/EPlast.WebApi/Models/UserPrecaution/UserPrecautionViewModel.cs]: Yes, it`s was mistake. I fixed it in the last commit

**Review by ostappppppp — APPROVED** [review]

**Review by hryhoriivoloshyn — APPROVED** [review]

**Pull request closed without merging**